### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by Appdevsoft** [Luehang/react-native-gallery-swiper#16]
How to use imageComponent. Please give me an some example

**Comment by Appdevsoft** [comment]
Please give me an example for imageComponent

**Comment by Luehang** [comment]
Hi,

You can use the `imageComponent` prop for a custom image.  An example is given below.  **Please give a star for this package if you like it.**  Thanks.

<br/>

---

<br/>

## :large_blue_diamond: Prop

Props | Description | Type | Default
------ | ------ | ------ | ------
`imageComponent` | Custom function to render your images. `(imageProps: { imageLoaded: Boolean, source: object, image: object, style: Array<object>, resizeMode: string, capInsets: object, onLoadStart: Function, onLoad: Function, ...extras }, imageDimensions: {width: number, height: number}, index: number) => React.Element` **index params included in Version \*1.3.1 update** | `Function` | `<Image/>` component

<br/>

---

<br/>

## :large_blue_diamond: Usage Example

```javascript
import React from "react";
import GallerySwiper from "react-native-gallery-swiper";
import { Image } from "react-native";

//...
render() {
    return (
        <GallerySwiper
            style={{ flex: 1, backgroundColor: "black" }}
            images={[
                { source: { uri: "https://luehangs.site/pic-chat-app-images/beautiful-beautiful-women-beauty-40901.jpg" } },
                { uri: "https://luehangs.site/pic-chat-app-images/animals-avian-beach-760984.jpg",
                    dimensions: { width: 1080, height: 1920 } },
                { URI: "https://luehangs.site/pic-chat-app-images/beautiful-blond-fishnet-stockings-48134.jpg" },
                { url: "https://luehangs.site/pic-chat-app-images/beautiful-beautiful-woman-beauty-9763.jpg" },
                { URL: "https://luehangs.site/pic-chat-app-images/attractive-balance-beautiful-186263.jpg" },
            ]}
            imageComponent={(imageProps) => {
               return (
                   <Image
                        {...imageProps}
                   />
               );
            }}
        />
    );
}
//...
```

<br/>

---

<br/>

From,

Lue Hang
